src/device-registry/utils/create-event.js

```diff
@@ -1105,7 +1107,7 @@ const createEvent = {
       external ? external : "noExternal"
     }_${airqloud ? airqloud : "noAirQloud"}_${
       airqloud_id ? airqloud_id : "noAirQloudID"
-    }`;
+    }_${lat_long ? lat_long : "noLatLong"}`;
```

Comment:
Updating the caching ID generation to cater for the case of querying measurements by `lat_long`